components/backgrounds/molecules.njk

```diff
@@ -0,0 +1,5 @@
+<div class="background-wrapper background-wrapper--molecules" role="presentation">
+  <div class="background background--molecules">
+    <img class="background__image background__image--molecules" src="./static/images/backgrounds/molecules.svg">
+  </div>
+</div>
```

Comment:
I wonder if it's enough to just use parent `background-wrapper--molecules` to modify children instead of creating modifiers. But of course it's better for performance to do it like this.

Reply:
yeah, could do that – this is just a result of evolving HTML 😜